pkg/vault/test/foundry/E2eErc4626Swaps.t.sol

```diff
@@ -349,9 +349,13 @@ contract E2eErc4626SwapsTest is BaseERC4626BufferTest {
             waUSDC.previewRedeem(balancesBefore.balances.vaultTokens[balancesBefore.waUsdcIdx]);
         uint256 vaultTotalUsdcAfter = balancesAfter.balances.vaultTokens[balancesAfter.usdcIdx] +
             waUSDC.previewRedeem(balancesAfter.balances.vaultTokens[balancesAfter.waUsdcIdx]);
+        console.log("vault usdc after: ", vaultTotalUsdcAfter);
+        console.log("vault usdc before: ", vaultTotalUsdcBefore);
+        // TODO: check sign; vaultTotalUsdcAfter - vaultTotalUsdcBefore = senderUsdcDelta, solve for usdcAfter
+        // to prevent underflows
```

Comment:
Is this TODO resolved? The signs are right because, in a Do/Undo operation, the `vaultTotalUsdcAfter` will always be bigger than `vaultTotalUsdcBefore` (The user should not get USDC or DAI out of the vault, or else the vault would be drained).
It worried me a bit that `vaultTotalUsdcAfter - vaultTotalUsdcBefore` threw an underflow, since the vault should never lose USDC tokens, but I imagine that happened with a small fee and a very small swap?